<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newAccountDialog</class>
 <widget class="QDialog" name="newAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="userVerifyLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="passwordVerifyLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>127</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="userID"/>
    </item>
    <item>
     <widget class="QLineEdit" name="userIDVerify"/>
    </item>
    <item>
     <widget class="QLineEdit" name="password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordVerify">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>270</width>
     <height>94</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Choose A Question</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mothers Maiden Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Street You Grew Up On</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Paternal Grandmother's First Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Maternal Grandmother's First Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Name Of Your First Pet</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>Choose A Question</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Your Favorite Color</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Make Of Your First Car</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Name of Your First Crush</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>First Name Of Your Oldest Sibling</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Name Of Your High School</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>272</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Enter User ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Re-enter User ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Enter Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Re-enter Password:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="verifyCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>verify account information</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>127</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLineEdit" name="securityAnsOne"/>
    </item>
    <item>
     <widget class="QLineEdit" name="securityAnsTwo"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
